forge-gui/res/cardsfolder/upcoming/white_suns_twilight.txt

```diff
@@ -0,0 +1,10 @@
+Name:White Sun's Twilight
+ManaCost:X W W
+Types:Sorcery
+A:SP$ GainLife | LifeAmount$ X | SubAbility$ DBToken | SpellDescription$ You gain X life. Create X 1/1 colorless Phyrexian Mite artifact creature tokens with toxic 1 and "This creature can't block." If X is 5 or more, destroy all other creatures. (Players dealt combat damage by a creature with toxic 1 also get a poison counter.)
+SVar:DBToken:DB$ Token | RememberTokens$ True | TokenAmount$ X | TokenScript$ c_1_1_a_phyrexian_mite_toxic_noblock | TokenOwner$ You | SubAbility$ DBWrath 
```

Comment:
are the mite tokens scripted?

Reply:
Mite token was done by #2213 also added to TypeList

Reply:
Nice!